services/pgnotifier/pgnotifier.go

```diff
@@ -111,20 +113,20 @@ func (notifier *PgNotifierT) triggerPending(topic string) {
 		time.Sleep(retriggerInterval)
 		stmt := fmt.Sprintf(`UPDATE %[1]s SET status='%[3]s',
 								updated_at = '%[2]s'
-								WHERE id IN  (
-								SELECT id
-								FROM %[1]s
-								WHERE status='%[3]s' OR status='%[4]s'
-								ORDER BY id
-								FOR UPDATE SKIP LOCKED
-								LIMIT %[5]v
-								);`,
+								WHERE ID IN (
```

Comment:
Is there some reason for this change? If not, let's use ID or id in these two lines to keep it consistent.